.github/workflows/browserslist-db-update.yml

```diff
@@ -8,8 +8,32 @@ on:
   workflow_dispatch:
 
 jobs:
+  close-existing-pr:
+    name: Close existing PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if branch exists
+        id: check-branch
+        continue-on-error: true
```

Comment:
I'm interested in this `continue-on-error` usage. Could it be a problem if, say, the request failed for some arbitrary reason and the `update-browserslist-database` job executed with the branch already existing?

Reply:
Updated the action to use a few additional checks before running the `update-browserslist-database` job.